src/components/GardensFilters.js

```diff
@@ -0,0 +1,68 @@
+import styled from 'styled-components'
+import { DropDown, GU, SearchInput, useLayout } from '@1hive/1hive-ui'
+import React from 'react'
+
+const GardensFilters = ({
+  itemsNetwork,
+  itemsSorting,
+  nameFilter,
+  networkFilter,
+  sortingFilter,
+  onNameFilterChange,
+  onNetworkFilterChange,
+  onSortingFilterChange,
+}) => {
+  const { layoutName } = useLayout()
+
+  return (
+    <div
+      css={`
+        display: flex;
+        justify-content: center;
+        width: ${layoutName === 'max' ? '65%' : 'auto'};
+        gap: ${1 * GU}px;
+        flex-wrap: wrap;
+      `}
+    >
+      <FilterItem>
+        <DropDown
```

Comment:
I would just remove this, since i alredy implemented it on the multichain support pr, for now we are going to just have one url per instance